<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>networkset</class>
 <widget class="QWidget" name="networkset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 255, 255,0.9);</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>598</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{font: 10pt &quot;Arial&quot;;font-weight:Bold;}
QPushButton{background-color: rgb(237, 212, 0);}
QLabel{background-color:rgba(0,0,0,0);}
QLabel{font: 11pt &quot;Arial&quot;;font-weight:Bold;}
QPushButton{border: none;}
</string>
   </property>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>47</y>
      <width>250</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(6,62,97,1);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>26</y>
      <width>153</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> color:rgba(6,62,97,1);</string>
    </property>
    <property name="text">
     <string>Remote Interface</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>47</y>
      <width>250</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(6,62,97,1);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>26</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> color:rgba(6,62,97,1);</string>
    </property>
    <property name="text">
     <string>Ethernet</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>622</x>
      <y>47</y>
      <width>250</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(6,62,97,1);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>622</x>
      <y>27</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> color:rgba(6,62,97,1);</string>
    </property>
    <property name="text">
     <string>GPS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>68</y>
      <width>160</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{padding-left:10px;text-align:left;}</string>
    </property>
    <property name="text">
     <string>LAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>105</y>
      <width>160</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{padding-left:10px;text-align:left;}</string>
    </property>
    <property name="text">
     <string>USB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>142</y>
      <width>160</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{padding-left:10px;text-align:left;}</string>
    </property>
    <property name="text">
     <string>Bluetooth</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>179</y>
      <width>160</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{padding-left:10px;text-align:left;}</string>
    </property>
    <property name="text">
     <string>Wi-fi</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
